frontend/src/products/Popup/PopupComponent.jsx

```
@@ -0,0 +1,86 @@
+            <h3 style={{ marginLeft: '5px' }}>{header}</h3>
+            <CloseOutlinedIcon 
+              style={{ color: '#98A2B3', fontSize: '20px', cursor: 'pointer' }} 
+              onClick={handleClose} 
+            />
+          </div>
+        )}
+        <div className={styles.popupContentContainer}>
+          <div
+            className={styles.popupContent}
+            style={{ color: textColor }}
+            dangerouslySetInnerHTML={{ __html: DOMPurify.sanitize(content) }}
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)